packages/terra-toolkit-docs/src/terra-dev-site/tool/terra-functional-testing/About.1.tool.mdx

```diff
@@ -64,59 +108,157 @@ Tests can be executed in any of the following supported form factors:
+
+Type: `string`
 
-The `Terra.describeTests` helper extends the `Terra.describeViewports` helper by additionally filter tests by locales and themes. All tests within this helper will be executed only if each of the form factors, locales, and themes listed in the options match those defined in the configurations for the current test run. If no form factors, locales, or themes are provided as options, then all tests under this helper will qualify to run in any form factor, locale, or theme. It is recommended to use `Terra.describeTests` over `Terra.describeViewports` because the same features in `Terra.describeViewports` can be achieved with `Terra.describeTests` by eliminating locales and themes and only specify the form factors. This helper accepts these arguments in the following order:
-- {string} title - The describe block title.
-- {Object} options - An object containing arrays of formFactors, locales, and themes that the block of tests will only qualify to execute in.
-- {function} - The block of tests to execute against each viewport.
+Required: `false`
 
+Default: `[data-terra-test-content] *:first-child`
 
-The below describe block qualifies to run because each of the form factors, locales, and themes provided to `describeTests` match the CLI options specified in the test runner.
-```sh
-terra wdio --formFactors tiny --locales en --themes terra-default-theme
+<Notice variant="important" ariaLevel="3">
+  This selector is used as the default screenshot selector for Terra.validates.element and Terra.validates.screenshot.
+</Notice>
+
+```js
+const { config } = require('@cerner/terra-functional-testing');
+
+config.serviceOptions = {
+  selector: '#root',
+};
+
+exports.config = config;
 ```
 
+### seleniumVersion
+
+Specifies the [docker selenium](https://github.com/SeleniumHQ/docker-selenium/tree/selenium-3) version used within the docker container.
+
+Type: `string`
+
+Required: `false`
+
+Default: `3.14.0-helium`
+
+<Notice variant="caution" ariaLevel="3">
+  The selenium version is only applied when running tests against a docker container. This option does not change the version on a remote selenium grid. Keep this in mind if your tests run against a remote selenium grid on a CI system.
+</Notice>
+
 ```js
-const testOptions = {
-  formFactors: ['tiny', 'huge'],
-  locales: ['en', 'fr'],
-  themes: ['terra-default-theme', 'orion-fusion-theme']
-}
+const { config } = require('@cerner/terra-functional-testing');
 
-Terra.describeTests('Describe tests', testOptions, () => {
-  it('should execute only if form factor is tiny or huge, locale is en or fr, and theme is terra-default-theme or orion-fusion-theme', () => {
+config.serviceOptions = {
+  seleniumVersion: '3.141.59-20210311',
+};
+
+exports.config = config;
+```
+
+## Test Utilities
+
+Test utilities are available to help write tests. These utilities are accessed using the `Terra` global object.
+
+## Describe Helpers
```

Comment:
Should this be a subheading under Test Utilities or its own level 2?

Reply:
I shifted everything one level:

https://github.com/cerner/terra-toolkit/pull/610/commits/36606851682a8a89e5cc76e7a6ec39436bd45d0c